reporter/html.js

```diff
@@ -747,13 +747,19 @@ QUnit.testDone(function( details ) {
 	}
 });
 
-if ( !defined.document || document.readyState === "complete" ) {
+if ( defined.document ) {
+	if ( document.readyState === "complete" ) {
+		if ( window.define && window.define.amd ) {
+			QUnit.config.autostart = false;
```

Comment:
Thanks for this! It's definitely a common gotcha for AMD users. I have one question, though: if it is an AMD environment, shouldn't auto-start be disabled regardless of the document's `readyState`?